content/docs/guides/chat_app.md

```diff
@@ -0,0 +1,279 @@
+---
+title: Build a Chat App
+date: 2023-03-01T14:00:00+10:00
+weight: 1
+---
+
+# Build a Chat App
+
+In this guide you will learn how to receive and send messages using Waku by building an app from scratch.
+If you want to learn how to add Waku to an existing app, check the [/quick_start] guide.
+
+This guide is kept succinct on purpose, check out the other [guides](/docs/guides) to learn more.
+
+## 1. Setup Project
+
+Setup a new npm package:
+
+```shell
+mkdir waku-app
```

Comment:
I am wondering if we can define this section by relying on `@waku/create-app`.

Reply:
create-app creates an example ready to use.
I think it may be interested to use create-app for a guide that modifies an existing example to implement a new feature.

Maybe something to consider when doing an encryption guide for example: we could start with a simple pre-exisitng example thanks to create-app and then add encryption to it.